lisp/init-javascript.el

```diff
@@ -57,7 +57,10 @@
     (add-hook 'xref-backend-functions #'xref-js2-xref-backend nil t))
   (with-eval-after-load 'js
     (define-key js-mode-map (kbd "M-.") nil)
-    (add-hook 'js-mode-hook 'sanityinc/enable-xref-js2))
+    (define-key js-ts-mode-map (kbd "M-.") nil)
```

Comment:
This would break in Emacs < 29 so a different formulation would be needed.

Reply:
Thanks for the reminder, I have fixed this problem, please take a look.